src/registry.rs

```diff
@@ -222,7 +222,28 @@ impl Registry {
 
 // Default registry for rust-prometheus.
 lazy_static! {
-    static ref DEFAULT_REGISTRY: Registry = Registry::default();
+    static ref DEFAULT_REGISTRY: Registry = {
+        let reg = Some(Registry::default());
+
+        match reg {
+            #[cfg(not(feature = "process"))]
+            Some(reg) => {
+                reg
+            },
+
+            // Register process collector.
+            #[cfg(feature = "process")]
+            Some(reg) => {
+                use process_collector::{get_pid, ProcessCollector};
```

Comment:
can we extract a function like new_process_collector for `process` and `none process` feature, so that we can use this function directly?